discovery-package/src/main/resources/zosmf-static-definition.yaml.template

```diff
@@ -0,0 +1,49 @@
+# Static definition for z/OSMF
+#
+# Once configured you can access z/OSMF via the API gateway:
+# curl -k -X GET -H "X-CSRF-ZOSMF-HEADER: *" https://${ZOWE_EXPLORER_HOST}:${GATEWAY_PORT}/api/v1/zosmf/info
+#
+services:
```

Comment:
I'm still not quite sure why there are 2 zosmf services defined, but recognise you haven't changed this! I wonder if in future it might be good to have a separate zosmf component defined instead of bundling with discovery, but would need to sort out the dependency management a bit first probably